README.md

```diff
@@ -79,6 +79,18 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
 
 The **default MSYS environment** is based on Cygwin, which **won't work**!
 
+*steps in order to build for windows:
+-Download msys2 then open the shell and type those commands respectively:
```

Comment:
For this list to render correctly, you need to add a space after each dash. Example:

```
- Download msys2 then ...
- Open msys2-mingw64 shell from ...
```

You can check the rendering of a markdown (.md) file on Github by clicking the three-dot menu on the right and choosing "view file".
```